### ISSUE TITLE
Add language on pootle_fs/xliff support

### ISSUE THREAD

**Issue by vasilvestre** [translate/pootle#5244]
Ubuntu 16.04
Pootle 2.8.0b3

Have succesfully created a pootle_fs project.
Based on xliff, I put a file with my GNU style "messages.<language_code>.<ext>"
I tried french and templates as language, both are taken, uploaded and I can translate.
But when I add any language, my RQworker get me this error in both language :

```
[2016-09-13T10:13:35]   default: pootle_project.forms.update_translation_project(<TranslationProject: /sq/messages/>, True, 'http://localhost:8000/') (cfeaa7eb-99ae-4e4f-bbab-3cfa0e96d436)
2016-09-13 10:13:55,524 INFO Scanning for new files in /sq/messages/
[2016-09-13T10:14:05]   AttributeError: 'NoneType' object has no attribute 'startswith'
Traceback (most recent call last):
  File "/home/valentin/dev/pootle/env/local/lib/python2.7/site-packages/rq/worker.py", line 568, in perform_job
    rv = job.perform()
  File "/home/valentin/dev/pootle/env/local/lib/python2.7/site-packages/rq/job.py", line 495, in perform
    self._result = self.func(*self.args, **self.kwargs)
  File "/home/valentin/dev/pootle/env/local/lib/python2.7/site-packages/pootle/apps/pootle_project/forms.py", line 45, in update_translation_project
    raise e
AttributeError: 'NoneType' object has no attribute 'startswith'
Traceback (most recent call last):
  File "/home/valentin/dev/pootle/env/local/lib/python2.7/site-packages/rq/worker.py", line 568, in perform_job
    rv = job.perform()
  File "/home/valentin/dev/pootle/env/local/lib/python2.7/site-packages/rq/job.py", line 495, in perform
    self._result = self.func(*self.args, **self.kwargs)
  File "/home/valentin/dev/pootle/env/local/lib/python2.7/site-packages/pootle/apps/pootle_project/forms.py", line 45, in update_translation_project
    raise e
AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Comment by unho** [comment]
@phlax Do you have any input on this one?

**Comment by phlax** [comment]
hopefully this will be fixed in #5899 @vasilvestre - thanks for reporting
